phantom-dsl/src/test/scala/com/outworkers/phantom/jdk8/tables/OptionalPrimitivesJdk8.scala

```diff
@@ -30,14 +30,22 @@ case class OptionalJdk8Row(
   localDateTime: Option[JdkLocalDateTime]
 )
 
+object Bla {
+
+  implicit val tpPrimitive: Primitive[OffsetDateTime] = OffsetDateTimeIsPrimitive()(Primitive[(Long, String)]())
+  implicit val zonedDt: Primitive[ZonedDateTime] = zonedDateTimePrimitive()(Primitive[(Long, String)]())
+}
+
+import Bla._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/flavian/phantom/pullRequest?prid=3695041)